modules/Core/module.php

```diff
@@ -923,21 +923,29 @@ public function onPageLoad(User $user, Pages $pages, Cache $cache, Smarty $smart
                         $full_ip = ['ip' => $default->ip . (is_null($default->port) ? '' : ':' . $default->port), 'pre' => $default->pre, 'name' => $default->name];
 
                         // Get query type
-                        $query_type = Util::getSetting('external_query') === '1' ? 'external' : 'internal';
+                        $query_type = Util::getSetting('external_query');
+                        if ($query_type == QueryType::INTERNAL) {
```

Comment:
Rather than storing these as numbers and then converting to/from strings a bunch, can we just store them in the `QueryType` class as string constants?